views/js/runner/plugins/security/preventCopy.js

```diff
@@ -140,19 +140,52 @@ define([
             var testRunner = this.getTestRunner();
             var prohibitedKeyFunc;
             var prohibitedKeyDebounce;
-
-            function prevent(event) {
-                if ($(event.target).closest('textarea, input, [contenteditable]')) {
-                    return;
+            var isIe = typeof window.clipboardData !== 'undefined';
+
+            function replaceSelection(target, newValue) {
+                const oldValue = target.value.toString().substring(target.selectionStart, target.selectionEnd);
+                const caretPosition = target.selectionStart;
+                const textBegin = target.value.substring(0, caretPosition);
+                const textEnd = target.value.substring(caretPosition + oldValue.length, target.value.length);
+                target.value = textBegin + newValue + textEnd;
+                target.selectionStart = caretPosition + newValue.length;
+                target.selectionEnd = caretPosition + newValue.length;
+                target.focus();
+            }
+            function onCopyCut(event) {
+                event.preventDefault();
```

Comment:
Would it work ,if I move preventDefault to the else branch and remove code from 160 to the 167 lines?